static/docker-compose.yml

```diff
@@ -135,6 +135,9 @@ services:
       LOG_LEVEL: "${LOG_LEVEL:-info}"
       REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
       REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
+    # Specifically needs the API_INTEGRATION_AUTH_CONFIG_NOTIFICATION_SERVICE generated in src/_configure_plextrac.sh
+    env_file:
+    - .env
```

Comment:
We actually don't want to include the `.env` @nicholasvrdoljak -- just the required envvars. Otherwise we get envvar creep and have to unwind which envvars are needed by which service. Simply add the required envvar to the `environment` section above.

Reply:
Done! See latest commit.